executors/sshexec/volume.go

```diff
@@ -155,6 +158,33 @@ func (s *SshExecutor) VolumeDestroyCheck(host, volume string) error {
 	return nil
 }
 
+func (s *SshExecutor) createVolumeOptionsCommand(volume *executors.VolumeRequest) []string {
+
+	godbc.Require(len(volume.Options) > 0)
+
+	commands := []string{}
+	var cmd string
+
+	// Go through all the Options and create volume set command
+	for _, volOption := range volume.Options {
+		if volOption != "" {
+			switch volOption {
+			case "encryption":
+				cmd = fmt.Sprintf("gluster --mode=script volume set %v server.ssl on", volume.Name)
```

Comment:
@humblec my comments were not about future enhancements, but about reasons why I would be reluctant to merge this patch as is.

I think we need to discuss the options (the names of the cmdline options, and whether to trigger the heketi-level options from the same heketi-cli cmdline option as the pass-through ones or from a different one) now, before merging anything, because otherwise we might create non-backward-compatible updates in the future.

I'd  be happy to merge a patch that does only the passing-through for now, and later, after design agreement has been reached, add the heketi-level convenience-wrappers.

But also for the passing-through we need to stop briefly and think about the naming. "--options" just seems to be too ambiguous. We should use something like --volume-options or --gluster-volume-options or --passthrough-options or so.

And the splitting up I'd still request. Now it would be 2 patches:
1. the prep patch with the clusters variable
2. the passthrough patch with the agreed name (currently proposed as --options)

Cheers - Michael